deployer/scripts/functions.sh

```diff
@@ -30,11 +30,23 @@ function setup_certificate {
 function handle_previous_deployment() {
   if [ "$mode" = "refresh" ]; then
     echo "Deleting any previous deployment (leaving route and PVCs)"
-    oc delete rc,svc,pod,sa,templates,secrets --selector="metrics-infra"
+    oc delete rc,svc,pod,sa,templates,secrets --selector="metrics-infra" --ignore-not-found=true
```

Comment:
isn't this also going to delete the deployer? I think if you left out "pod" here it would be fine as the only standalone pod is the deployer.